index.html

```diff
@@ -407,12 +407,12 @@ <h4 class="panel-title">
           </tbody>
           <tbody ng-repeat="bufferline in bufferlines">
             <tr class="bufferline">
-              <td class="time">
+              <td class="time" ng-show="activeBuffer().showBufferLineTimes">
```

Comment:
This should use one-time bindings (`::` prefix) because it never changes once set. It should also be a class of the `bufferlines` div like `withNicklist` that affects all `td.time` and `td.prefix` beneath it, because it affects all lines of a buffer.

Reply:
I did it with a class, I didn't get the one time binding in there because each time the buffer chages this should be updated.

Reply:
Yeah once it's on the table object it needs to be a watcher, but it's only a single one so no problem